Xamarin.Forms.Xaml/PruneIgnoredNodesVisitor.cs

```diff
@@ -17,12 +20,21 @@ public void Visit(ElementNode node, INode parentNode)
 			foreach (var propertyKvp in node.Properties)
 			{
 				var propertyName = propertyKvp.Key;
-				var propertyValue = (propertyKvp.Value as ValueNode)?.Value as string;
-				if (propertyValue == null)
+				if (!((propertyKvp.Value as ValueNode)?.Value is string propertyValue))
 					continue;
 				if (!propertyName.Equals(XamlParser.McUri, "Ignorable"))
 					continue;
-				(parentNode.IgnorablePrefixes ?? (parentNode.IgnorablePrefixes = new List<string>())).AddRange(propertyValue.Split(','));
+				var prefixes = propertyValue.Split(',').ToList();
```

Comment:
The `,` is the incorrect splitter here.
https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/mc-ignorable-attribute
> Multiple prefixes can be specified in the attribute, using one or more white-space characters as the separator, for example: mc:Ignorable="ignore1 ignore2".

Reply:
good catch